app/models/ticket.rb

```diff
@@ -60,12 +60,13 @@ def self.total_price_user(conference, user)
     tickets.inject(0){ |sum, ticket| sum + (ticket.amount_paid * ticket.quantity) }
   end
 
-  def tickets_sold
-    ticket_purchases.paid.sum(:quantity)
+  def tickets_turnover_total(id)
+    tickets = TicketPurchase.where(ticket_id: id)
+    tickets.inject(0){ |sum, ticket| sum + (ticket.amount_paid * ticket.quantity) }
```

Comment:
I would have only one method for this line, and then calling it for different tickets, as this way you don't need to duplicate the line, and at least in one on the cases you already have the tickets.